docs/01-overview/README.md

```diff
@@ -1,46 +1,14 @@
 ---
-title: What is Kyma?
+title: Kyma components
 ---
 
-Kyma is an opinionated set of Kubernetes-based modular building blocks, including all necessary capabilities to develop and run enterprise-grade cloud-native applications.
-It is the open path to the SAP ecosystem supporting business scenarios end-to-end.
+Classic Kyma offered a fixed set of preconfigured components whose development rhythm is synchronized and determined by the release schedule.
 
-![overview](./assets/modular-kyma.png)
+See the list of Kyma components that have not been transformed into modules:
```

Comment:
```suggestion
See the list of Kyma components that have not been transformed into modules yet:
```
Do we want to indicate that they will?

Reply:
I am note sure because we have UI on the list that I guess won't.

Reply:
I will discuss migrating the content with Valentin and create a separate PR as this is out of scope here :)

Reply:
Good point, agree